docs/tutorials/explore-available-content.md

```diff
@@ -97,38 +97,33 @@ Then you can query the catalog by using `curl` commands and the `jq` CLI tool to
 3. Return list of packages that support `AllNamespaces` install mode and do not use webhooks:
 
     ``` terminal
-    curl -k https://localhost:8443/catalogs/operatorhubio/api/v1/all | jq -cs '[.[] | select(.schema == "olm.bundle" and (.properties[] | select(.type == "olm.csv.metadata").value.installModes[] | select(.type == "AllNamespaces" and .supported == true)) and .spec.webhookdefinitions == null) | .package] | unique[]'
+    curl -k https://localhost:8443/catalogs/operatorhubio/api/v1/metas?schema=olm.bundle | jq -cs '[.[] | select(.properties[] | select(.type == "olm.csv.metadata").value.installModes[] | select(.type == "AllNamespaces" and .supported == true) and .spec.webhookdefinitions == null) | .package] | unique[]'
```

Comment:
Need a note here that it's dependent on catalogs using the new `olm.csv.metadata` bundle metadata record (which requires an explicit opt-in from folks with a flag, for e.g. `opm render --migrate-level bundle-object-to-csv-metadata`). 
And this record will likely not be present on anything migrated from sqlite that isn't a channel head, since the migration _should_ be pruning metadata from non-channel-heads. 

Maybe change the case of the example to something like

> Return list of packages where the channel head versions use `olm.csv.metadata` metadata format, support `AllNamespaces`...